pom.xml

```diff
@@ -6,7 +6,7 @@
     <parent>
```

Comment:
Mode `<generateSupportingFiles>false</generateSupportingFiles>` to `configuration ` instead of `configOptions`